.github/workflows/ios.yml

```diff
@@ -0,0 +1,84 @@
+name: IOS
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+    paths:
+      - 'go/**'
+      - 'go.*'
+      - '**.go'
+      - 'js/**'
+      - '.github/workflows/ios.yml'
+  pull_request:
+    paths:
+      - 'go/**'
+      - 'go.*'
+      - '**.go'
+      - 'js/**'
+      - '.github/workflows/ios.yml'
+
+jobs:
+  build:
+    name: Build for iOS
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - name: Cache Go modules
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-
+
+      - name: Cache Node modules
+        uses: actions/cache@v2
+        with:
+          path: |
+            js/node_modules
+            js/packages/*/node_modules
+          key: ${{ runner.OS }}-node-${{ hashFiles('js/**/yarn.lock') }}
+          restore-keys: ${{ runner.OS }}-node-
+
+      - name: Cache Ruby modules
+        uses: actions/cache@v2
+        with:
+          path: js/vendor
+          key: ${{ runner.OS }}-bundle-${{ hashFiles('js/Gemfile.lock') }}
+          restore-keys: ${{ runner.OS }}-bundle-
+
+      - name: Build the Archive
+        working-directory: js
+        env:
+          SHAKEBUGS_API_ID: ${{ secrets.SHAKEBUGS_API_ID }}
+          SHAKEBUGS_API_SECRET: ${{ secrets.SHAKEBUGS_API_SECRET }}
+        run: |
+          echo "SHAKE_API_ID=$SHAKEBUGS_API_ID\nSHAKE_API_SECRET=$SHAKEBUGS_API_SECRET" > .env
+          make vendor/bundle deps packages/go-bridge/ios/Frameworks/Bertybridge.framework packages/berty-app/ios/Pods/Manifest.lock
+          xcodebuild archive \
+                -workspace packages/berty-app/ios/Berty.xcworkspace \
+                -scheme yolo \
+                -configuration Release \
+                -sdk iphoneos \
+                -archivePath /tmp/Berty.xcarchive \
+                CODE_SIGN_IDENTITY="" CODE_SIGNING_ALLOWED=NO
+          tar -jcvf ${{ github.workspace }}/Berty\ Yolo.app.tar.bz2 /tmp/Berty.xcarchive/Products/Applications/Berty\ Yolo.app
+
+      - name: Upload the Archive
+        uses: actions/upload-artifact@v2
+        with:
+          path: ${{ github.workspace }}/Berty Yolo.app.tar.bz2
```

Comment:
```suggestion
          path: /tmp/Berty.xcarchive/Products/Applications/Berty\ Yolo.app
          name: berty-yolo.app
```

Reply:
the path change is because, github action is already zipping the uploaded artifact

the name is to ease the recognition of the artifact kind by yolo

Reply:
Ok 👍